docs/Using-Ethsigner/Getting-Started.md

```diff
@@ -7,10 +7,17 @@ EthSigner requires a V3 Keystore key file and a password file.
 
 ## Prerequisites 
 
-[EthSigner](../Installation/Install-Binaries.md)
-[Pantheon](https://docs.pantheon.pegasys.tech/en/stable/Installation/Install-Binaries/)
-[Node.js](https://nodejs.org/en/download/)
-[web3.js](https://github.com/ethereum/web3.js/)
+* [EthSigner](../Installation/Install-Binaries.md)
+
```

Comment:
Are these blank lines made on purpose ? There is no need of them for a list. Only one before the list is required.

Reply:
Fixed